clients/lsp-elixir.el

```diff
@@ -100,6 +100,39 @@ finding the executable with `exec-path'."
   :group 'lsp-elixir
   :type 'file)
 
+(defcustom lsp-elixir-enable-test-lenses t
+  "Suggest Tests."
+  :type 'boolean
+  :group 'lsp-elixir
+  :package-version '(lsp-mode . "7.1"))
+
+(defun lsp-elixir--build-test-command (argument)
+  "Builds the test command from the ARGUMENT."
+  (let ((test-name (gethash "testName" argument))
```

Comment:
using gethash is not allowed. All the access to the data structures that come from the language server should happen either by defined `lsp-interface` or by using `lsp-get`. This will allow us to migrate to using plists instead of hashtables. You may refer to rust-analyzer for example: https://github.com/yyoncho/lsp-mode/blob/headerline-errors/lsp-protocol.el#L278

Reply:
thanks, will have a look.

Reply:
@yyoncho I changed to lsp-get, but will figure out how lsp-interface works tomorrow if it might be a better fit.

Reply:
`lsp-get` is good enough for non-core components given the fact that this structure will be accessed only in a few places.

As a side note, most likely vscode supports running the tests in the debugger, right? We can implement that as well.

Reply:
Do you mean dap-mode? I have not really worked with it before, but can have a look. Ideally i would not want to click to run tests, but this is the baseline work for future capabilities it seems.

Reply:
> Do you mean dap-mode?

yes.

> Ideally i would not want to click to run tests, but this is the baseline work for future capabilities it seems.

There is `lsp-avy-lens`. And then you will be able to re-run with `M-x recompile`.